modules/log6x-input-spec-filter-namespace-container.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-input-spec-filter-namespace-container_{context}"]
+= Filtering application logs at input by including or excluding the namespace or container name
+
+You can include or exclude the application logs based on the namespace and container name by using the `input` selector.
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+* You have created a `ClusterLogForwarder` custom resource (CR).
+* You must have a `serviceAccount` in the same namespace in which you create the `ClusterLogForwarder`. Additionally, for this `ClusterLogForwarder` you must have the `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles. For more information see link:https://docs.openshift.com/container-platform/4.16/observability/logging/log_collection_forwarding/log-forwarding.html#log-collection-rbac-permissions_log-forwarding[Authorizing log collection RBAC permissions].
```

Comment:
This cannot be used here. 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

OCP does not allow hard links to internal content. 
OCP requires xrefs for internal content. 
OCP does not allow xrefs in modules, only assemblies.